src/main/java/com/researchspace/dcd/utils/DigitalCommonsDataAdapterUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.researchspace.dcd.utils;
+
+import com.researchspace.repository.spi.License;
+import com.researchspace.repository.spi.LicenseDef;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Collections;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DigitalCommonsDataAdapterUtils {
+
+  public static List<License> getDigitalCommonsDataLicenses() throws MalformedURLException {
+    License license = new License();
+    license.setLicenseDefinition(new LicenseDef(new URL("https://creativecommons.org/publicdomain/zero/1.0/"), "CC-0"));
+    return Collections.singletonList(license);
+  }
```

Comment:
Seems a bit unnecessary to have an extra class with one method? I'd just put the code in `DigitalCommonsDataRepository`.

Reply:
removed